faster_raster.go

```diff
@@ -110,18 +110,20 @@ type Rasterizer struct {
 	stopCompleted      chan struct{}
 }
 
-func NewRasterizer(filename string) *Rasterizer {
+// NewRasterizer returns a configured Rasterizer for a given filename,
+// which is able to process parallelRequests requests simultaneously.
```

Comment:
That's not right. They are not parallel. It's a buffer, as originally named.
```suggestion
// which uses a buffered channel of rasterBufferSize to queue requests for the rasterizer.
```